mlir/docs/Bindings/Python.md

```diff
@@ -1035,7 +1035,7 @@ class ConstantOp(_ods_ir.OpView):
     ...
 ```
 
-expects `value` to be a `TypedAttr` (e.g., `IntegerAttr` or `FloatAttr`). 
+expects `value` to be a `TypedAttr` (e.g., `IntegerAttr` or `FloatAttr`).
```

Comment:
e.g., here also.

Same everywhere.

Reply:
I exported these changes here: https://github.com/llvm/llvm-project/pull/121791, so I update this PR once the other one with cosmetics will be merged and these changes will disappear.
My IDE automaticallly will remove trailing space etc so prefer to wait a bit...